learn/core_concepts/indexes.md

```diff
@@ -145,3 +145,15 @@ Typo tolerance is a built-in feature that helps you find relevant results even w
 You can update the typo tolerance settings using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update typo tolerance endpoint](/reference/api/settings.md#update-typo-tolerance-settings).
 
 [Learn more about typo tolerance.](/learn/configuration/typo_tolerance.md)
+
+## Swapping indexes
+
+Suppose you want to deploy a new version of your `movies` index in production. This `movies` index is where your users are currently making search requests. The changes you made are in `movies_new`. To sync `movies_new` with Meilisearch in production, you will use index swapping.
+
+This means that the schema, settings, and task history of  `movies` will be swapped with the schema, settings, and task history of `movies_new` **without any downtime for the search clients**. The task history of `enqueued` tasks is not modified.
+
+Once swapped, your users will still be making search requests to the `movies` index but it will contain the data of `movies_new`. You can delete `movies_new` after the swap or keep it in case something goes wrong and you want to swap back.
```

Comment:
What about documents? They are not mentioned explicitly, but this phrase

> Once swapped, your users will still be making search requests to the `movies` index but **it will contain the data of `movies_new`**.

makes me think that swapping indexes also swaps the documents.